product_docs/docs/eprs/7/01_introduction/03_certified_supported_versions.mdx

```diff
@@ -4,8 +4,8 @@ title: "Certified and supported product versions"
 
 You can use the following database product versions with Replication Server:
 
--   PostgreSQL versions 12, 13, 14, 15, and 16
--   EDB Postgres Advanced Server versions 12, 13, 14, 15, and 16
+-   PostgreSQL versions 12, 13, 14, 15, 16 and 17
```

Comment:
We maintain last 5 major versions of PostgreSQL, so with the addition of v17, v12 should be dropped.